From c503834519d21973323980850431101f90e663ef Mon Sep 17 00:00:00 2001
From: Jan Tojnar <jtojnar@gmail.com>
Date: Mon, 25 Mar 2019 15:23:59 +0100
Subject: [PATCH] doc: Use namespace correctly

The files declared xmlns:db but did not use the db namespace at all.
They did not define the default namespace at all, which coincidentally
worked with Docbook 4 stylesheets, making them think the files were
written in Docbook 4.

I fixed the namespaces of the documents and switched to the correct
Docbook 5 stylesheets.

Signed-off-by: Jan Tojnar <jtojnar@gmail.com>
---
 doc/arping.xml      | 2 +-
 doc/clockdiff.xml   | 2 +-
 doc/custom-html.xsl | 2 +-
 doc/custom-man.xsl  | 2 +-
 doc/meson.build     | 2 +-
 doc/ninfod.xml      | 2 +-
 doc/ping.xml        | 2 +-
 doc/rarpd.xml       | 2 +-
 doc/rdisc.xml       | 2 +-
 doc/tftpd.xml       | 2 +-
 doc/tracepath.xml   | 2 +-
 doc/traceroute6.xml | 2 +-
 12 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/doc/arping.xml b/doc/arping.xml
index ed00af4..711718f 100644
--- a/doc/arping.xml
+++ b/doc/arping.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.arping">
 
   <refentryinfo>
diff --git a/doc/clockdiff.xml b/doc/clockdiff.xml
index dea05c8..24d1943 100644
--- a/doc/clockdiff.xml
+++ b/doc/clockdiff.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.clockdiff">
 
   <refentryinfo>
diff --git a/doc/custom-html.xsl b/doc/custom-html.xsl
index c391535..a952c5f 100644
--- a/doc/custom-html.xsl
+++ b/doc/custom-html.xsl
@@ -2,7 +2,7 @@
 
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/docbook.xsl"/>
+<xsl:import href="http://docbook.sourceforge.net/release/xsl-ns/current/html/docbook.xsl"/>
 <!--
   - The docbook stylesheet injects empty anchor tags into generated HTML, identified by an auto-generated ID.
   - Ask the docbook stylesheet to generate reproducible output when generating (these) ID values.
diff --git a/doc/custom-man.xsl b/doc/custom-man.xsl
index 058f495..ba9697d 100644
--- a/doc/custom-man.xsl
+++ b/doc/custom-man.xsl
@@ -5,7 +5,7 @@
                 extension-element-prefixes="exsl"
                 version="1.0">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"/>
+<xsl:import href="http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl"/>
 
 <xsl:template name="top.comment" />
 
diff --git a/doc/meson.build b/doc/meson.build
index 50dcb4e..5052857 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -49,7 +49,7 @@ xsltproc_args = [
 
 if xsltproc.found()
 	testrun = run_command([xsltproc, '--nonet',
-		'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl'])
+		'http://docbook.sourceforge.net/release/xsl-ns/current/manpages/docbook.xsl'])
 	xsltproc_works = testrun.returncode() == 0
 else
 	warning('No docbook stylesheet found for generating man pages')
diff --git a/doc/ninfod.xml b/doc/ninfod.xml
index 1177a21..081d470 100644
--- a/doc/ninfod.xml
+++ b/doc/ninfod.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.ninfod">
 
   <refentryinfo>
diff --git a/doc/ping.xml b/doc/ping.xml
index 691887a..b5cea9c 100644
--- a/doc/ping.xml
+++ b/doc/ping.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.ping">
 
   <refentryinfo>
diff --git a/doc/rarpd.xml b/doc/rarpd.xml
index 0785d56..65d6367 100644
--- a/doc/rarpd.xml
+++ b/doc/rarpd.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.rarpd">
 
   <refentryinfo>
diff --git a/doc/rdisc.xml b/doc/rdisc.xml
index 8aa8898..ab1c673 100644
--- a/doc/rdisc.xml
+++ b/doc/rdisc.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.rdisc">
 
   <refentryinfo>
diff --git a/doc/tftpd.xml b/doc/tftpd.xml
index 8f51c3d..4a1b948 100644
--- a/doc/tftpd.xml
+++ b/doc/tftpd.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.tftpd">
 
   <refentryinfo>
diff --git a/doc/tracepath.xml b/doc/tracepath.xml
index d0a9a12..d1bdcd8 100644
--- a/doc/tracepath.xml
+++ b/doc/tracepath.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.tracepath">
 
   <refentryinfo>
diff --git a/doc/traceroute6.xml b/doc/traceroute6.xml
index c52d296..5a72784 100644
--- a/doc/traceroute6.xml
+++ b/doc/traceroute6.xml
@@ -1,4 +1,4 @@
-<refentry xmlns:db="http://docbook.org/ns/docbook" version="5.0"
+<refentry xmlns="http://docbook.org/ns/docbook" version="5.0"
 xml:id="man.traceroute6">
 
   <refentryinfo>
-- 
2.20.1

